edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -550,10 +550,10 @@ You may receive questions regarding on-the-spot registration at the competition.
 1. Competitors must not be accepted on the day of the competition under any circumstances unless on-the-spot registration for the competition is enabled.
 2. If registration is full, on-the-spot registrations _must not_ be accepted on the day of the competition.
 
-Some regions have created alternative waiting list formats that can allow for maximizing the amount of competitors who get to compete and to mitigate no-shows. The Extra Registration Requirements must clearly outline these policies in order to implement them. Here are some examples of acceptable formats we have approved in the past when the process was explicitly outlined in the Extra Registration Requirements:
+Some regions have created alternative waiting list formats that can allow for maximizing the number of competitors who get to compete and to mitigate no-shows. The Extra Registration Requirements must clearly outline these policies in order to implement them. Here are some examples of acceptable formats that have been approved in the past when the process was explicitly outlined in the Extra Registration Requirements:
 
 1. Competitors on the waiting list may be added in the order of the waiting list after the waiting list deadline if the conditions are specified.
-2. Competitors attempting to register on the spot when registration is full may be added on the day of the competition in order of arrival when on-the-spot registration is enabled if the conditions are specified.
+2. Competitors attempting to register on the spot when registration is full may be added on the day of the competition in order of arrival when on-the-spot registration is enabled, if the conditions are specified.
```

Comment:
The comma here should be removed.

Reply:
I thought it's needed, because this part of the sentence could be removed and it would still be grammatically correct

Reply:
I'm pretty sure you'd need another word (I forget the type of word, but it's a type of word that connects the two phrases).

I'd usually see it if you moved the last part to the front.

"If the conditions  are specified, competitors attempting to register on the spot when registration.....registration is enabled."

My understanding is you would need something else after the comma to have it in the form you have currently.  I forget the word for it.